### PR TITLE
Add link to setup guide

### PR DESCRIPTION
**Pull request by MadLittleMods**
.

 - Add link to setup guide
    - Should we put this in line with the readme instead of linking?
 - Some other cleanup
    - Markdown formatting
    - Removed trailing whitespace (automatically done by editor and can revert)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,34 @@
...
-<img src="https://i.imgur.com/QpEYrRo.png"/>
-Makes Splitscreen easy for Halo:Online V0.6+
-	
-	#Features
 
-	No need for multiple installs
+Makes Splitscreen easy for Halo:Online v0.6+
 
-	Automatic controller enabling/port handling
+For a full setup guide, see https://gist.github.com/MadLittleMods/dd0aa40c62fe1e200e8aa4f8b4fed605
```
Comment (MadLittleMods) [inline README.md:5]: Main update is here

**Review by DeathDealer33 — APPROVED** [review]

**Comment by MadLittleMods** [comment]
@DeathDealer33 What's holding up the actual merge?

**Comment by DeathDealer33** [comment]
@MadLittleMods Im new to github and im not sure how things work:) Thats the hold up, i manually added the link to your setup guide.

**Comment by MadLittleMods** [comment]
@DeathDealer33 If you're happy with the changes, just press the green "Merge" button below

**Comment by MadLittleMods** [comment]
Thanks for the merge @DeathDealer33

**Pull request merged**